imports/lib/publications/pendingAnnouncementsForSelf.ts

```diff
@@ -0,0 +1,5 @@
+import TypedPublication from './TypedPublication';
+
+export default new TypedPublication<void>(
+  'PendingAnnouncements.publications.forSelf'
```

Comment:
convention bikeshedding: I'm not sure we need to put `.publications` in here?  These are already `TypedPublication` and the context these strings will pop up in will be "a publication name".  Was there a reason you added the additional layer here, or a pattern of use you were imagining?

Reply:
Possibly not. I did the same thing in `TypedMethod`, and was basically riffing off of the style that [mdg:validated-method seems to recommend](https://github.com/meteor/validated-method#using-simpleschema)

Honestly, naming in general is a little more annoying here than it was with `TypedMethod`, because in most cases the thing you're subscribing to is a noun, and those nouns already exist as names. With `TypedMethod` at least it's usually a noun+verb. (You can see in these examples how I have to bend over a bit to avoid naming things in ways that conflict with other files - e.g. this isn't just `pendingAnnouncements`, even though that's what we called the old subscription)